openstack-k8s-operators/ovn-operator
# [Kuttl] Add test for OVN db cluster pod deletions

test steps:

1. stand up 3 replicas ovn db (nb) cluster.
2. confirm pods are all up.
3. confirm that the pods established the mesh.
4. delete all pods.
5. check that new pods are respawned.
6. check that they re-established the mesh with the correct number of pods,
and all connections are good.

related Jira: OSPRH-6135
